src/officehours_api/serializers.py

```diff
@@ -36,27 +36,12 @@ class Meta:
 
 
 class NestedMyMeetingSerializer(serializers.ModelSerializer):
-    line_place = serializers.SerializerMethodField(read_only=True)
     backend_metadata = serializers.JSONField(read_only=True)
 
     class Meta:
         model = Meeting
         fields = ['id', 'line_place', 'agenda', 'assignee', 'backend_type', 'backend_metadata', 'created_at']
 
-    def get_line_place(self, obj):
-        i = 0
-        in_line = False
-        meetings = obj.queue.meeting_set.order_by('id')
-        for i in range(0, len(meetings)):
-            if self.context['user'] in meetings[i].attendees.all():
-                in_line = True
-                break
-
-        if in_line:
-            return i
-        else:
-            return None
-
```

Comment:
Moved this into the `Meeting` model (and refactored the contents)

Reply:
Seems like the right move.